backend/src/main/resources/db/migration/V1_0_11__Update_Information_About_Profile.sql

```diff
@@ -0,0 +1,29 @@
+ALTER TABLE credentials
+    ADD COLUMN phone_number NUMERIC(9,0) NULL;
+ALTER TABLE credentials
+    ADD COLUMN birth_date DATE NULL;
+
+ALTER TABLE locations
+    ADD COLUMN zip_code CHAR(5) NOT NULL;
+ALTER TABLE locations
+    ADD COLUMN island VARCHAR(15) NOT NULL;
```

Comment:
ValueObject?

Reply:
Hi Ibai, I don't think an object value is needed for those values. If you think otherwise please tell me.

Reply:
Ok, it's done.